internal/app/init/pkg/system/conditions/files.go

```diff
@@ -33,7 +33,7 @@ func (filename file) Wait(ctx context.Context) error {
 }
 
 func (filename file) String() string {
-	return fmt.Sprintf("for file %q to exist", string(filename))
+	return fmt.Sprintf("file %q to exist", string(filename))
```

Comment:
this was leading to duplicate `for`, as master description looks like `Waiting for [conditions.str().join(',')]"